gis4wrf/bootstrap.py

```diff
@@ -80,6 +80,7 @@
     if not DATA_HOME:
         DATA_HOME = os.path.join(os.path.expanduser('~'), '.local', 'share')
 INSTALL_PREFIX = os.path.join(DATA_HOME, 'gis4wrf', 'python')
+LOG_PATH = os.path.join(INSTALL_PREFIX, 'pip.log')
```

Comment:
we are eventually moving to `pathlib` only. Can you change this?

Reply:
That should be a separate PR.

Reply:
That's fine -- it will be taken care by #15.